bucket/kyverno-cli.json

```
@@ -0,0 +1,27 @@
+{     
```

```suggestion
{
```

Unnecessary whitespaces.